src/plugins/stdioproc/README.md

```diff
@@ -0,0 +1,175 @@
+- infos = Information about the stdioproc plugin is in keys below
+- infos/author = Klemens Böswirth <k.boeswirth+git@gmail.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = pregetstorage getstorage procgetstorage postgetstorage presetstorage setstorage
+- infos/status = maintained unittest shelltest experimental
+- infos/metadata =
+- infos/description = one-line description of stdioproc
+
+## Introduction
+
+This plugin spawns a new process with a user-defined executable and delegates all operations to the new process.
+
+## Usage
+
+Set the config key `app` and the arrays `args/#`, `env/#` to the path of an executable, the arguments that shall be passed and the environment variables to be set.
+
+```
+kdb mount test.dump /tests/stdioproc specload 'app=/usr/bin/pluginproc' 'args=#1' 'args/#0=--load-plugin' 'args/#1=myplugin'
```

Comment:
Is this ready to review? Is this really specload?

Reply:
> Is this ready to review?

There aren't any tests which check that running a Java Plugin via `stdioproc` actually works. I've checked it manually and I'm currently working on automatic tests. Also some parts of the docs still need updates. Otherwise this PR should be done.

> Is this really specload?
It is somewhat similar, but
1. serves an entirely different purpose
2. `specload` uses only one-way communication, while `stdioproc` is two-way
3. `specload` only sends a single keyset, `stdioproc` has a more complex protocol

Reply:
> Is this really specload?

I missed that the line literally says "specload". That's a copy paste error. It's fixed in the next commit.